packages/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -527,6 +534,8 @@ export const Chart: React.FunctionComponent<ChartProps> = ({
     ...(name && { name: `${name}-${(legendComponent as any).type.displayName}` }),
     orientation: legendOrientation,
     theme,
+    ...(legendDirection === 'rtl' && { dataComponent: <ChartPoint transform="translate(40)" /> }),
+    ...(legendDirection === 'rtl' && { labelComponent: <ChartLabel direction="rtl" dx="10" /> }),
```

Comment:
We would want to clone these components instead of overriding a component

Reply:
👍 

With a custom data/label component the positioning might be slightly off, but a user could leave out the `legendDirection` prop and pass their own transforms and offsets instead.

Reply:
Does `translate(40)` work for different lengths of text? 

Is this what flips the legend symbol (i.e., ChartPoint) to the right-side?